java/compaction-status-store/src/test/java/sleeper/compaction/status/testutils/DynamoDBCompactionTaskStatusStoreTestBase.java

```diff
@@ -75,9 +75,24 @@ protected static CompactionTaskStatus finishedTaskWithDefaults() {
                 defaultFinishTime().toEpochMilli()).build();
     }
 
+    protected static CompactionTaskStatus finishedTaskWithDefaultsAndDurationInSecondsNotAWholeNumber() {
+        Instant finishTime = Instant.parse("2022-09-22T14:00:14.500Z");
+        return startedTaskWithDefaultsBuilder().finished(
+                CompactionTaskFinishedStatus.builder()
+                        .addJobSummary(jobSummary(finishTime)),
+                finishTime.toEpochMilli()).build();
+    }
+
     private static CompactionJobSummary defaultJobSummary() {
+        return jobSummary(defaultFinishTime());
+    }
+
+    private static CompactionJobSummary jobSummary(Instant finishTime) {
         Instant jobStartedUpdateTime = Instant.parse("2022-09-22T14:00:04.000Z");
-        Instant jobFinishTime = Instant.parse("2022-09-22T14:00:14.000Z");
+        return createJobSummary(jobStartedUpdateTime, finishTime);
+    }
+
+    private static CompactionJobSummary createJobSummary(Instant jobStartedUpdateTime, Instant jobFinishTime) {
```

Comment:
It should just be jobStartedTime rather than jobStartedUpdateTime. Could the default times be extracted to their own functions like defaultJobStartTime() and defaultJobFinishTime(), and could they be put next to the task default times defaultStartTime and defaultFinishTime please?